core/psh/pshapp/auth.c

```diff
@@ -139,12 +139,12 @@ static int psh_login(int argc, char **argv)
 		return 1;
 
 	sleep(2);
-	return -1;
+	return 0;
```

Comment:
maybe in separate PR - return codes should be sorted out. See for example discussion in `ping`: https://github.com/phoenix-rtos/phoenix-rtos-utils/pull/70#issuecomment-866750946

We should probably return `0` on success and some positive values in case of errors - in `main()` function of the applet. For internally used functions we can have different semantics, but it should be consistent (eg. (0 -> success, negative -> error) OR (1 -> succes, 0 -> fail (no matter what))

Reply:
Yes. The returns in `auth` go against the return values rules. As for now i will change them in `auth` just to not forget about them. But sorting out return codes is a thing in psh in I will take care later of it (as there is also issue for that)